NullPointerException

Hi @franktip. Thanks for an amazing tool! I was trying to update the data for PLDI (see #32) with the most recent incarnations of the conference.

* I was able to populate the PCs using the `pc-from-researchr.py` script. That seemed to work fine.
* I then downloaded the html file from DBLP using `wget`. That seemed to produce an HTML file with all of the papers.
* But when I ran `./gradlew run` at the top level, I got a `NullPointerException`.

Is this expected? Has DBLP's format perhaps changed?

The command-line output is below. The problematic file seems to be [this one](https://github.com/pcminer-tools/pcminer/blob/452b400f0dfeadd84c96051a1a9474c0c2604e20/data/PLDI/2021/pldi2021.html).

```
% ./gradlew run

> Task :run
ICSE2013 ICSE2014 ICSE2015 ICSE2012 ICSE1995 ICSE2008 ICSE2001 ICSE2006 ICSE2007 ICSE2000 
ICSE2009 ICSE2017 ICSE2010 ICSE2019 ICSE2018 ICSE2011 ICSE2016 ICSE1997 ICSE1999 ICSE1998 
ICSE1996 ICSE2005 ICSE2002 ICSE2003 ICSE2004 OOPSLA2013 OOPSLA2014 OOPSLA2015 OOPSLA2012 OOPSLA1995 
OOPSLA2008 OOPSLA2001 OOPSLA2006 OOPSLA2007 OOPSLA2000 OOPSLA2009 OOPSLA2010 OOPSLA2011 OOPSLA2016 OOPSLA1997 
OOPSLA1999 OOPSLA1998 OOPSLA1996 OOPSLA2005 OOPSLA2002 OOPSLA2003 OOPSLA2004 PPoPP2013 PPoPP2014 PPoPP2015 
PPoPP2012 PPoPP1993 PPoPP1995 PPoPP2008 PPoPP2001 PPoPP2006 PPoPP2007 PPoPP2009 PPoPP2010 PPoPP2011 
PPoPP1997 PPoPP1990 PPoPP1999 PPoPP1991 PPoPP2005 PPoPP2003 ISMM2013 ISMM2014 ISMM2015 ISMM2012 
ISMM2008 ISMM2006 ISMM2007 ISMM2000 ISMM2009 ISMM2010 ISMM2011 ISMM2002 ISMM2004 TFP2013 
TFP2014 TFP2015 TFP2012 TFP2008 TFP2001 TFP2006 TFP2007 TFP2000 TFP2009 TFP2017 
TFP2010 TFP2019 TFP2020 TFP2018 TFP2011 TFP2016 TFP1999 TFP2005 TFP2003 TFP2004 
ECOOP2013 ECOOP2014 ECOOP2015 ECOOP2012 ECOOP2008 ECOOP2001 ECOOP2006 ECOOP2007 ECOOP2000 ECOOP2009 
ECOOP2017 ECOOP2010 ECOOP2019 ECOOP2018 ECOOP2011 ECOOP2016 ECOOP1997 ECOOP1999 ECOOP1998 ECOOP1996 
ECOOP2005 ECOOP2002 ECOOP2003 ECOOP2004 ASE2013 ASE2014 ASE2015 ASE2012 ASE2008 ASE2001 
ASE2006 ASE2007 ASE2009 ASE2017 ASE2010 ASE2018 ASE2011 ASE2016 ASE2005 ASE2002 
ASE2003 ASE2004 ICFP2013 ICFP2014 ICFP2015 ICFP2012 ICFP2008 ICFP2001 ICFP2006 ICFP2007 
ICFP2000 ICFP2009 ICFP2010 ICFP2011 ICFP2016 ICFP1997 ICFP1999 ICFP1998 ICFP1996 ICFP2005 
ICFP2002 ICFP2003 ICFP2004 ESOP2013 ESOP2014 ESOP2015 ESOP2012 ESOP1988 ESOP1986 ESOP1994 
ESOP1992 ESOP2008 ESOP2001 ESOP2006 ESOP2007 ESOP2000 ESOP2009 ESOP2010 ESOP2011 ESOP1990 
ESOP1999 ESOP1998 ESOP1996 ESOP2005 ESOP2002 ESOP2003 ESOP2004 FSE2013 FSE2014 FSE2015 
FSE2012 FSE2008 FSE2001 FSE2006 FSE2007 FSE2000 FSE2009 FSE2017 FSE2010 FSE2019 
FSE2018 FSE2011 FSE2016 FSE1997 FSE1999 FSE1998 FSE2005 FSE2002 FSE2003 FSE2004 
Haskell2013 Haskell2014 Haskell2015 Haskell2012 Haskell2008 Haskell2006 Haskell2007 Haskell2009 Haskell2017 Haskell2010 
Haskell2019 Haskell2020 Haskell2018 Haskell2011 Haskell2016 Haskell2005 Haskell2003 Haskell2004 PACMPL2017 OOPSLA2019 
OOPSLA2021 POPL2020 OOPSLA2018 POPL2013 POPL2014 POPL2015 POPL2012 POPL1995 POPL2008 POPL2001 
POPL2006 POPL2007 POPL2000 POPL2009 POPL2017 POPL2010 POPL2011 POPL2016 POPL1997 POPL1999 
POPL1998 POPL1996 POPL2005 POPL2002 POPL2003 POPL2004 PLDI2013 PLDI2014 PLDI2015 PLDI2012 
PLDI1995 PLDI2008 PLDI2001 PLDI2006 PLDI2007 PLDI2000 PLDI2009 PLDI2017 PLDI2010 PLDI2019 
PLDI2021 Exception in thread "main" java.lang.NullPointerException
        at pcminer.PCMiner$1.visitTag(PCMiner.java:162)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:469)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at org.htmlparser.tags.CompositeTag.accept(CompositeTag.java:478)
        at pcminer.PCMiner.scanConference(PCMiner.java:155)
        at pcminer.PCMiner.scan(PCMiner.java:113)
        at pcminer.PCMiner.process(PCMiner.java:71)
        at pcminer.PCMiner.main(PCMiner.java:62)

> Task :run FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':run'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk-14.0.2.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
2 actionable tasks: 1 executed, 1 up-to-date

```

--- a/src/pcminer/PCMiner.java
+++ b/src/pcminer/PCMiner.java
@@ -159,6 +159,8 @@ public class PCMiner {
               public void visitTag(Tag tag) {
                 if (tag instanceof BulletList) {
                   BulletList bl = (BulletList) tag;
+                  if (bl.getChildren() == null)
+                      return;
                   for (int i = 0; i < bl.getChildren().size(); i++) {
                     Node child = bl.childAt(i);
                     if (child instanceof TagNode) {

--- a/src/pcminer/PCMiner.java
+++ b/src/pcminer/PCMiner.java
@@ -159,8 +159,7 @@ public class PCMiner {
               public void visitTag(Tag tag) {
                 if (tag instanceof BulletList) {
                   BulletList bl = (BulletList) tag;
-                  if (bl.getChildren() == null)
-                      return;
+                  if (bl.getChildren() == null) return;
                   for (int i = 0; i < bl.getChildren().size(); i++) {
                     Node child = bl.childAt(i);
                     if (child instanceof TagNode) {

